client/state/selectors/get-simple-payments.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { get, find } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+
+/**
+ * Get all Simple Payment or the one specified by `simplePaymentId`
+ *
+ * @param {Object} state           Global state tree
+ * @param {int}    siteId          Site which the Simple Payment belongs to.
+ * @param {int}    simplePaymentId The ID of the Simple Payment to get. Optional.
+ * @return {Array|Object|null}     Array of Simple Payment objects or an object if `simplePaymentId` specified.
+ */
+export default createSelector(
+	( state, siteId, simplePaymentId = null ) => {
```

Comment:
I prefer to set it to `null` since it's clear that it's an optional parameter that way. Also, if we don't set a default value and call `getSimplePayments( state, siteId )`, phpstorm/editors will complain that the third parameter was not specified. On the other hand, if we give a default value, editors won't complain.